Automattic/vip-go-mu-plugins
# Send prolonged rate limiting messages to logstash

## Description

Currently we only send prolonged rate limiting messages to Slack. This should also go to logstash for debugging purposes.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Set the rate limiting thresholds for Search queries and indexing low.
2. Trigger ratelimiting. There should be nothing related sent to logstash.
3. Apply PR.
4. Trigger ratelimiting. A logstash entry should be created.
